presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/TransformQuantifiedComparisonApplyToScalarApply.java

```diff
@@ -164,7 +164,7 @@ private PlanNode rewriteQuantifiedApplyNode(ApplyNode node, QuantifiedComparison
                     node.getId(),
                     context.rewrite(node.getInput()),
                     subqueryPlan,
-                    Assignments.identity(minValue, maxValue),
+                    Assignments.identity(minValue, maxValue, countAllValue, countNonNullValue),
```

Comment:
The tests in the following commits break without that. So it needs to stay in this PR and there are tests for that. Plus, it's only needed due to the requirements IterativeOptimizer introduces